### PR TITLE
Fix Linux build

### PR DESCRIPTION
**Pull request by TotalCaesar659**
Fix Linux build after https://github.com/grumpycoders/pcsx-redux/pull/215 and https://github.com/grumpycoders/pcsx-redux/pull/217.

### CODE REVIEW

File: src/gui/gui.cc [hunk 1]
```diff
@@ -325,7 +325,7 @@ void PCSX::GUI::endFrame() {
     bool changed = false;
 
     if (m_fullscreenRender) {
-        ImTextureID texture = reinterpret_cast<ImTextureID*>(&m_offscreenTextures[m_currentTexture]);
+        ImTextureID texture = reinterpret_cast<ImTextureID*>(m_offscreenTextures[m_currentTexture]);
```
Comment (nicolasnoble) [inline src/gui/gui.cc:328]: Woops ;-)

Comment (TotalCaesar659) [reply src/gui/gui.cc:328]: Do you mean I need to revert this?

Comment (nicolasnoble) [reply src/gui/gui.cc:328]: No, no, it was a problem from the original change you've done. That's the proper fix here.

Comment (TotalCaesar659) [reply src/gui/gui.cc:328]: Yes, I haven't seen it then. Here it is some docs if you need: https://github.com/ocornut/imgui/issues/1840

**Pull request merged**